region/lowernorfair/east/Red Kihunter Shaft.json

```diff
@@ -548,6 +550,65 @@
       ],
       "flashSuitChecked": true
     },
+    {
+      "link": [2, 3],
+      "name": "G-Mode",
+      "entranceCondition": {
+        "comeInWithGMode": {
+          "mode": "any",
+          "morphed": true
+        },
+        "comesThroughToilet": "any"
+      },
+      "requires": [
+        "h_canArtificialMorphMovement",
+        "canOffScreenMovement",
+        {"or": [
+          "h_canArtificialMorphBombThings",
+          {"and": [
+            "canXRayStandUp",
+            "canPartialFloorClip",
+            "canCeilingClip",
+            {"heatFrames": 350},
+            {"or": [
+              "canTrickyGMode",
+              {"heatFrames": 200}
+            ]}
+          ]}
+        ]},
+        {"or": [
+          "ScrewAttack",
+          {"enemyDamage": {"enemy": "Kihunter (red)", "type": "contact", "hits": 1}}
+        ]},
+        "h_HeatedGModeOffCameraDoor"
+      ],
+      "exitCondition": {
+        "leaveNormally": {}
+      },
+      "unlocksDoors": [
+        {
+          "types": ["missiles"],
+          "requires": [{"heatFrames": 50}]
+        },
+        {"types": ["super"], "requires": []},
+        {
+          "types": ["powerbomb"],
+          "requires": [{"heatFrames": 110}]
+        }
+      ],
+      "flashSuitChecked": true,
+      "note": [
+        "Overload PLMs by rolling through the camera scroll block just right of the bomb blocks,",
+        "then roll through the morph tunnel and use a bomb boost or Spring Ball to escape the tunnel and get to the door.",
+        "Without a way to boost out of the tunnel, exit G-mode and use a partial floor clip then ceiling clip to jump out of the tunnel.",
```

Comment:
Its better described as a 2 turn xray climb.  The partial floor clip would need the blocks gone, but using xray to stand up makes them solid again.